tools/Nethermind.Tools.Kute/ResponseTracer/FileResponseTracer.cs

```diff
@@ -7,9 +7,9 @@ namespace Nethermind.Tools.Kute.ResponseTracer;
 
 public class FileResponseTracer : IResponseTracer
 {
-    private readonly string _tracesFilePath;
+    private readonly string? _tracesFilePath;
 
-    public FileResponseTracer(string tracesFilePath)
+    public FileResponseTracer(string? tracesFilePath)
```

Comment:
This parameter should never be null

Reply:
In this case, I will make it required so the app will throw when the flag is missing.